src/framework/pages/utils/TimeManager.java

```diff
@@ -0,0 +1,66 @@
+package framework.utils;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
```

Comment:
This class was delete the other class time manager, It class has code repeated, and as I understand you don't need this class to set meetings, Also you can't have your code hard-coded like in   getMoreThirtyMinutes, please could you review that 
ASAP